specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4366,13 +4366,24 @@
       "description": "settings for security policy patterns to match",
       "type": "object",
       "properties": {
+        "IsProfileLevel": {
+          "description": "Flag to signal whether the WAF policy association is at the default Profile level.",
+          "type":"boolean"
```

Comment:
This should be an enum. Booleans will generally fail validation since it's not extensible.